pkg/yamltemplate/filetests/ytt-library/assert/assert-equals-err-args.yml

```diff
@@ -0,0 +1,11 @@
+#@ load("@ytt:assert", "assert")
+#@ load("@ytt:json", "json")
```

Comment:
```suggestion
```

This module is not used in this test.

Let's check the other test files as well to make sure they don't include unused code.